src/modules/common/utils/textTruncator.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+const textTruncator = (text: string, numOfWordsAllowed: number): string => {
+  const ellipsis = '...'
+  const splitTextToWords = text.split(' ')
```

Comment:
Better naming would be just `words`

Reply:
I agree. Should I create a new PR for this since the current one is merged?